### PR TITLE
Cancel requestAnimationFrame loop for ending youtube vids

### PR DESCRIPTION
**Pull request by meandavejustice**
- youtubes iframe player ends differently than normal videos,
it wants to end at one second before it's duration. This commit
addresses that. It is a hack but works.

- fixed issue where youtube videos were not showing replay/close
ui on video end.

- add back close method for replay/close ui view

- refs #184 (not a complete fix)

### CODE REVIEW

**Comment by 6a68** [comment]
looking now

File: components/player-view.js [hunk 1]
```diff
@@ -26,9 +27,13 @@ module.exports = React.createClass({
       progress: currentTime / window.AppData.duration
     });
 
-    if (currentTime >= window.AppData.duration) {
-      window.AppData.playing = false;
-      window.AppData.playedCount++;
+    // TODO: the "-1" is a hack to force YouTube embeds to do what we want.
```
Comment (pdehaan) [inline components/player-view.js]: Is this the #184 issue? If so, we may want to add the bug number to the TODO comment for future context.

Comment (meandavejustice) [inline components/player-view.js]: good thinking @pdehaan will do.

**Comment by 6a68** [comment]
LGTM :+1:

**Pull request merged**